vendor/stmicroelectronics/nucleo-wl55jc1.yaml

```diff
@@ -0,0 +1,74 @@
+name: NUCLEO-WL55JC1
+description: NUCLEO-WL55JC1 high-band development board with temperature sensor and controllable LEDs
+hardwareVersions:
+  - version: '1.0'
+    numeric: 1
+firmwareVersions:
+  - version: '1.0.0'
+    numeric: 1
+    hardwareVersions:
+      - '1.0'
+    profiles:
+      EU863-870:
+        id: nucleo-wl55jc-profile-eu868
+        lorawanCertified: true
+        codec: nucleo-wl55jc-codec
```

Comment:
This is a codec for a specific example, I suppose, right?

Would it be useful to name this firmware version then? This is what we did here as well: https://github.com/TheThingsNetwork/lorawan-devices/blob/9bdc8c5e96a70246d18ff03a898937d212a23046/vendor/the-things-products/the-things-uno.yaml#L6

Reply:
Thats right, this codec is aligned on LoRaWAN_End_Node projects: [STM32CubeWL LoRaWAN_End_Node](https://github.com/STMicroelectronics/STM32CubeWL/tree/main/Projects/NUCLEO-WL55JC/Applications/LoRaWAN/LoRaWAN_End_Node)

But I'm quite surprised, because with the exception of the the-things-uno.yaml file, all the examples currently offered on the master branch use the version field with a version string, and not to refer to a project name.
On the other hand, I agree that there is a missing field to reference the firmware to use (name, url, ...).

Reply:
True, but for development kits a real version number is not super useful.

We don't enforce any string format on version, so I'm fine using a (versioned) project name.